res_json.c

```diff
@@ -11,7 +11,7 @@
  * the project provides a web site, mailing lists and IRC
  * channels for your use.
  *
- * This program is free software, distributed under the terms of
+ * This program is ast_free software, distributed under the terms of
```

Comment:
:) oh the pitfalls of search-replace!

Reply:
Yes, I've note this too, but as it is a comment, this is irrelevant. All "free" ocorrencies was checked before do pull request.